CMakeLists.txt

```diff
@@ -1,27 +1,41 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.5)
 project(pcl_conversions)
 
-find_package(catkin REQUIRED COMPONENTS pcl_msgs roscpp sensor_msgs std_msgs cmake_modules)
+if(NOT WIN32)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14 -Wall -Wextra")
+endif()
 
-find_package(PCL REQUIRED QUIET COMPONENTS common)
-find_package(Eigen REQUIRED)
+find_package(ament_cmake REQUIRED)
 
-include_directories(include ${catkin_INCLUDE_DIRS} ${PCL_COMMON_INCLUDE_DIRS} ${Eigen_INCLUDE_DIRS})
+find_package(Boost REQUIRED)
```

Comment:
Unfortunately, libpcl itself uses boost all over the place.  Thus, we need to have boost both in CMakeLists.txt and in package.xml, since it truly is a dependency here (you cannot successfully link without boost).